### ISSUE TITLE
Keyboard trap "bumpers" cause aria-hidden has no focusable content rule false positive [6cfa84]

### ISSUE THREAD

**Issue by WilcoFiers** [act-rules/act-rules.github.io#1824]
This issue was raised on axe-core:
https://github.com/dequelabs/axe-core/issues/3430

Here's a codepen showing the issue: https://codepen.io/pankhurinigam/pen/BawPRbQ

Since this is a real-world false positive, I think we need an example to cover it.

**Comment by Jym77** [comment]
Is this fixed by #1819 / #1820 ? (seems like the "focus sentinel" is the same as the "trap bumper")

**Comment by WilcoFiers** [comment]
This issue was fixed a while back. Not sure when, but it's definitely resolved.
